rdbi-recipes/src/test/java/com/lithium/dbi/rdbi/recipes/cache/SimpleRedisCacheTest.java

```diff
@@ -0,0 +1,110 @@
+package com.lithium.dbi.rdbi.recipes.cache;
+
+import com.codahale.metrics.MetricRegistry;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.lithium.dbi.rdbi.Handle;
+import com.lithium.dbi.rdbi.RDBI;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import redis.clients.jedis.JedisPool;
+
+import java.util.concurrent.Callable;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.AssertJUnit.assertEquals;
```

Comment:
I don't know much about testng, but this looks fishy to me.  Should we not be importing ```org.testng.Assert. assertEquals``` instead?